scenario-definitions.yaml

```diff
@@ -13,6 +13,8 @@ machines:
       HDDSIZEGB: "20"
       QEMUCPU: host
       QEMURAM: "2048"
+      MODE: install
+      INSTALL_ONLY: '1'
```

Comment:
These vars aren't used anymore in the code

Reply:
thx,